setup/security/ssl.md

```diff
@@ -15,10 +17,55 @@ Spinnaker instance. That is, any requests between...
 
 * any other client and Gate
 
-> **Info**: Many operators like to get authentication working before adding
+## Network configurations
+
+**Warning**: Many operators like to get authentication working before adding
 HTTPS, but experience bears out that the transition is not smooth. We recommend
 you implement at least a temporary SSL solution **first**.
 
+
+
+
+## Load Balancer-terminated SSL
+
+A common practice is to offload SSL-related bits to outside of the server in
+question. This is fully supported in Spinnaker, but it does affect the
+authentication configuration slightly. See your [authentication
+method](/setup/security/authentication/) for specifics.
+
+![SSL terminated at load balancer](/setup/security/authentication/network-arch/lb-ssl-termination.png)
+
+During the certain authentication workflows, Gate makes an intelligent guess on how to assemble a URI to
```

Comment:
"During certain authentication workflows" or "During the authentication workflows" — probably not both ("the certain")